src/PrestaShopBundle/Resources/config/services/adapter/backup.yml

```diff
@@ -1,12 +1,23 @@
 services:
   _defaults:
-    public: true
+    public: false
+    autowire: true
+
+  PrestaShop\PrestaShop\Adapter\Backup\DatabaseBackupCreator:
+  PrestaShop\PrestaShop\Adapter\Backup\BackupRepository:
+  PrestaShop\PrestaShop\Adapter\Backup\BackupRemover:
 
   prestashop.adapter.backup.database_creator:
-    class: 'PrestaShop\PrestaShop\Adapter\Backup\DatabaseBackupCreator'
+    alias: 'PrestaShop\PrestaShop\Adapter\Backup\DatabaseBackupCreator'
+    public: false
```

Comment:
```suggestion
    public: true
```

Reply:
You can't change the default visibility of a service in a minor version, at least not on the alias